bucket.go

```diff
@@ -111,12 +112,13 @@ func (client *Client) ListBuckets() ([]*Bucket, error) {
 	return buckets, err
 }
 
+// ListTestsInput represents the input to ListTests func
 type ListTestsInput struct {
 	BucketName string
 	Count      int
 }
 
-// ListBuckets lists all buckets for an account
+// ListTests lists all buckets for an account
```

Comment:
~all buckets for an account~/all tests for a bucket

Reply:
Done